docassemble/ALAffidavitOfIndigency/data/questions/affidavit_body.yml

```
@@ -488,6 +501,24 @@ validation code: |
   if len([fee for fee in fees if fees[fee].waive]) < 1:
     validation_error("You need to choose at least one.", field="fees['Other cost'].waive")
 ---
+template: what_fee_amount_should_i_write_template
+subject: |
+  What amount should I write in?
+content: |
+  If you are not sure of the exact amount yet, it is OK to leave it blank. You can 
+  also write your best guess. For common fees, the court clerk may know the amount and 
+  can fill it in for you.
+
+  You may also be able to find the fee by visiting the court's website:
+  
+  * [List of filing fees in Massachusetts](https://www.mass.gov/info-details/uniform-schedule-of-fees).
+
+  It usually costs about $40 to serve a document on the opposing party, either by using the Sherrif or
+  a constable.
+
+  If you are asking for something special to your circumstances, it may help the court decide
+  if you research the exact fee amount.
```

Slight rephrasing suggestion:
```suggestion
  If you are asking for something special to your circumstances, researching the exact fee amount
  yourself may help the court decide.
```